boards/arm/stm32f3_disco/stm32f3_disco_E.overlay

```diff
@@ -31,4 +31,4 @@
         spi-max-frequency = <1000000>;
         label = "I3G4250D";
     };
-};
\ No newline at end of file
+};
```

Comment:
Please move this change to the commit `boards: arm: add i3g4250d gyro to stm32f3_disco@E`

Reply:
> Please move this change to the commit `boards: arm: add i3g4250d gyro to stm32f3_disco@E`

sure, done